### PR TITLE
Sphinx docs

### PR DESCRIPTION
**Pull request by jonasvdd**
TODO:

- [x] Remove pdoc from files / pyproject.toml / ...
- [x] rewriting limitation section as dynamic graph construction now is supported! 🔥 
- [x] review all the docs! 💪🏼 
- [x] look for CI-CD ⚙️ 
- [ ] how to embed prior releases (tags) into docs? 🤯

### CODE REVIEW

File: tests/test_figure_resampler_selenium.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def test_multiple_tz(driver, multiple_tz_figure):
         raise e
     finally:
         proc.terminate()
+    multiple_tz_figure.stop_server()
```
Comment (jvdd) [inline tests/test_figure_resampler_selenium.py:74]: Why not in the finally clause (gets always executed)

Comment (jonasvdd) [reply tests/test_figure_resampler_selenium.py:74]: good catch!

**Review by jvdd — APPROVED** [review]

File: tests/test_figure_resampler_selenium.py (hunk 1 above)
Comment (jvdd) [reply tests/test_figure_resampler_selenium.py:74]: Will fix this myself

**Comment by jvdd** [comment]
LGMT! :rocket:

**Pull request merged**